Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -899,24 +903,34 @@ NonnullRefPtr<ClassExpression> Parser::parse_class_expression(bool expect_class_
         bool is_static = false;
         bool is_constructor = false;
         bool is_generator = false;
+        bool is_async = false;
         auto method_kind = ClassMethod::Kind::Method;
 
         if (match(TokenType::Semicolon)) {
             consume();
             continue;
         }
 
+        if (match(TokenType::Async)) {
+            consume();
+            is_async = true;
+        }
+
```

Comment:
Another edge case which is not covered now is a non async method called `async`.
```js
class C {
    async() {
    }
}
```
See the `name = "static"` examples just below here. (Although not covering this case for now is fine)

Reply:
Fixed